application/run.sh

```diff
@@ -18,22 +18,19 @@ chown -R www-data:www-data \
 echo 'sleeping a random number of seconds up to 9 to avoid migration runs clash'
 sleep $[ ( $RANDOM % 10 ) ]s
 
-if [[ -z "${APP_ID}" ]]; then
-  /data/yii migrate --interactive=0
-
-  if [[ ! -z $RUN_TASK ]]; then
-    ./yii $RUN_TASK
-    exit $?
-  fi
-
-  apache2ctl -k start -D FOREGROUND
+if [[ ${PARAMETER_STORE_PATH} ]]; then
+  config="config-shim --path $PARAMETER_STORE_PATH"
+elif [[ ${APP_ID} ]]; then
```

Comment:
It might be nice to stick to one syntax or the other. I think I prefer this to `! -z`, but I had to double check if they were equivalent.

Reply:
Good call